tests/test_pruner.py

```diff
@@ -30,22 +30,9 @@ def test_pruner():
         print(" \n GREEDY")
         radius = gaussian_heuristic(r) * 1.6
         print("pre-greedy radius %.4e" % radius)
-        tt = clock()
+        tt = process_time()
         pruning =Pruning.run(radius, overhead, r, 200, flags=Pruning.ZEALOUS, metric="solutions")
-        print("Time %.4e"%(clock() - tt))
-        print("post-greedy radius %.4e" % radius)
```

Comment:
Why did you delete all this?

Reply:
It was the same as the part before? The part for the greedy pruning test was there twice with the same constant parameters, at least it looks like that to me.

Reply:
Ah, indeed. Thanks